firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionServiceImplTest.java

```diff
@@ -506,12 +511,33 @@ private AlertDialog assertAlertDialogShown() {
   }
 
   @Test
-  public void signOutTester_setsSignInStatusFalse() throws InterruptedException {
+  public void signOutTester_setsSignInStatusFalse() {
     firebaseAppDistribution.signOutTester();
-    awaitTermination(lightweightExecutor);
+
     verify(mockSignInStorage).setSignInStatus(false);
   }
 
+  @Test
+  public void signOutTester_unsetsCachedNewRelease()
+      throws InterruptedException, FirebaseAppDistributionException, ExecutionException {
+    Task<AppDistributionReleaseInternal> setCachedNewReleaseTask =
+        firebaseAppDistribution.getCachedNewRelease().set(TEST_RELEASE_NEWER_AAB_INTERNAL);
+
+    // Confirm that the cached new release is initially set
+    awaitTask(setCachedNewReleaseTask);
+    assertThat(setCachedNewReleaseTask.getResult()).isEqualTo(TEST_RELEASE_NEWER_AAB_INTERNAL);
+
+    // Sign out the tester
+    firebaseAppDistribution.signOutTester();
+    awaitAsyncOperations(lightweightExecutor);
```

Comment:
If I understand this change correctly, this should no be needed

Reply:
Ah you're right, great catch!